.github/workflows/mdbook_cicd.yml

```diff
@@ -1,7 +1,12 @@
 name: Spec MDBook CI
 
-on: [pull_request]
-
+on:
+  push:
+    branches: [main]
+  create:
+    tags: 
+      - '^\d{8}\.\d+'
+  pull_request:
 jobs:
```

Comment:
there is a trigger when  a tag is created, tag should be dated and only tags that adhere to defined regex will be triggered